apps/condo/domains/miniapp/schema/B2BAppAccessRightSet.test.js

```diff
@@ -1360,4 +1468,73 @@ describe('B2BApp permissions for service user', () => {
 
         })
     })
+
+    describe('B2BAccessTokens domain', () => {
+
+        let app
+        let context
+        let serviceUser
+        let scopedRightSet
+        let accessRightSet
+
+        beforeAll(async () => {
+            const [organization] = await registerNewOrganization(user)
+            const [newServiceUser] = await registerNewServiceUserByTestClient(support)
+            serviceUser = await makeLoggedInClient({ email: newServiceUser.email, password: newServiceUser.password });
+            [app] = await createTestB2BApp(support);
+            [context] = await createTestB2BAppContext(support, app, organization, { status: 'Finished' })
+            const [right] = await createTestB2BAppAccessRight(support, serviceUser.user, app);
+            [accessRightSet] = await createTestB2BAppAccessRightSet(support, app, {
+                canReadB2BAccessTokens: true,
+                canManageB2BAccessTokens: true,
+            });
+            [scopedRightSet] = await createTestB2BAppAccessRightSet(support, app, { type: 'SCOPED' })
+            await updateTestB2BAppAccessRight(support, right.id, { accessRightSet: { connect: { id: accessRightSet.id } } })
+        })
+
```

Comment:
describe('Service user')

Reply:
Tests needed to test serviceUser restrictions
And in every operation with model, which permissions restricts user, are executed by service user. Why would I need describe block with him?